scripts/pcmping

```diff
@@ -0,0 +1,167 @@
+        res = send(pkt, iface=interface, verbose=False)
+        have_received, delay = receive(sockets, SELECT_TIMEOUT, exp_pkt, exp_socket, time_sent) 
+        if have_received:
+            return pkt, exp_pkt
+    return None, None
+
+def get_portchannel(interface):
+    configdb = swsssdk.ConfigDBConnector()
+    configdb.connect()
+    portchannels = configdb.get_table("PORTCHANNEL")
+    for key, value in portchannels.iteritems():
+        if interface in value['members']:
+            return key, value
+    print ("Interface %s is not a portchannel member. Please Check!" % interface)
+    sys.exit(1)
+
+def get_portchannel_ipv4(portchannel_name):
+    configdb = swsssdk.ConfigDBConnector()
+    configdb.connect()
+    config = configdb.get_config()
+    portchannel_interfaces = config["PORTCHANNEL_INTERFACE"]
+    for key in portchannel_interfaces.keys():
+        pc, ip = key
+        ip = ip.split("/")[0]
+        if pc == portchannel_name and ipaddress.IPAddress(ip).version == 4:
+            return ip
+    print ("Portchannel %s doesn't have IPV4 address. Please Check!" % portchannel)
+    sys.exit(1)
+   
+def create_socket(interface, portchannel):
+    # create sockets
+    try:
+        sockets = []
+        for iface in portchannel['members']:
+            s = socket.socket(socket.AF_PACKET, socket.SOCK_RAW, socket.htons(ETH_P_IP))
+            s.bind((iface, 0))
+            s.settimeout(RCV_TIMEOUT)
+            if iface == interface:
+                exp_socket = s
+            sockets.append(s)
+    except:
+        print "Unable to create socket. Check your permissions"
+        sys.exit(1)
+    return sockets, exp_socket
+
+def print_statistics(interface, total_count, recv_count):
+    print "\n--- %s ping statistics ---" % interface
+    print ("%d packets transmitted, %d received, %0.2f%% packet loss" % (total_count, recv_count, (total_count-recv_count)*1.0/total_count*100))
+
+def main():
+    parser  = argparse.ArgumentParser(description='Check portchannel member link state',
+                                      version='1.0.0',
+                                      formatter_class=argparse.RawTextHelpFormatter,
+                                      epilog=""" 
+    Example:
+    pcmping -p Ethernet8 -ip 10.10.10.10
+    pcmping -p Ethernet8 -ip 10.10.10.10 -c 5
+    pcmping -p Ethernet8 -ip 10.10.10.10 -c 5 -t 20
+"""
+    )
+
+    parser.add_argument('-p', '--port', type=str, required=True, help='Portchannel member interface')
+    parser.add_argument('-ip','--decapip', type=str, required=True, help='Neighbor decap ip (usually same as neighbor loopback ip)') 
+    parser.add_argument('-c', '--count', type=int, help='Number of decap packets to be sent.', default=0)
+    parser.add_argument('-t', '--trial', type=int, help='Maximum trial times to create decap packet.\nPlease increase the value as the number of portchannel members increases', default=20)
+    args = parser.parse_args()
+    
```

Comment:
can you change the packet size? I believer to test fcs error, we need to use large packet size (jumbo)